libcore/src/geometry/Building.cpp

```diff
@@ -27,10 +27,13 @@
 #include "Building.h"
 
 #include "general/OpenMP.h"
+#include "geometry/Line.h"
+#include "geometry/Point.h"
 #include "geometry/SubRoom.h"
 #include "geometry/Wall.h"
+#include "geometry/helper/CorrectGeometry.h"
 
-#include <chrono>
+#include <algorithm>
```

Comment:
remove unnecessary header

Reply:
I fixed the includes for `Building.cpp` but header and other files must be fixed in separate PR. We should check for other tools like IWYU.